app/controllers/tickets_controller.rb

```diff
@@ -49,7 +49,7 @@ def edit_person_validators
     ticket.metadata.tickets_edit_person_fields.each do |edit_person_field|
       case edit_person_field[:field_name]
       when TicketsEditPersonField.field_names[:dob]
-        dob_validation_issues = ResultsValidators::PersonsValidator.dob_validations(Date.parse(edit_person_field[:new_value]))
+        dob_validation_issues = ResultsValidators::PersonsValidator.dob_validations(Date.parse(edit_person_field[:new_value]), { name: ticket.metadata.wca_id })
```

Comment:
If you like my other suggestion about the method signature, then this can become...
```suggestion
        dob_validation_issues = ResultsValidators::PersonsValidator.dob_validations(Date.parse(edit_person_field[:new_value]), name: ticket.metadata.wca_id)
```

Reply:
Sidenote: You should probably extract the `Date.parse` into its own line, helps with debugging :)

Reply:
Done